help/start-a-call.md

```diff
@@ -6,7 +6,7 @@
 
 1. Click the **video-camera** (<i class="fa fa-video-camera"></i>) icon in the
 bottom left corner of the compose box. This will insert a link like
-**[Click to join video call]\(https://meet.jit.si/123456789)** into the
```

Comment:
Let's remove the extraneous dash in "video-camera" while we're here.

Reply:
Actually, a bit more of an update on these instructions:

{!start-composing.md!}

1. Click the **video camera** (<i class="fa fa-video-camera"></i>) at the bottom of 
 the compose box. This will insert a **[Join video call.]** link into your message.

1. Send the message.